packages/application-shell/src/components/navbar/main-navbar.styles.ts

```diff
@@ -143,40 +145,41 @@ const TooltipContainer = styled.div<{
 `;
 
 const Tooltip = styled.div`
-  padding: var(--spacing-10) calc(var(--spacing-20) + var(--spacing-10));
+  padding: ${uiKitDesignTokens.spacing10}
+    calc(${uiKitDesignTokens.spacing20} + ${uiKitDesignTokens.spacing10});
   display: -webkit-box;
   -webkit-line-clamp: 2;
   -webkit-box-orient: vertical;
-  border-radius: var(--border-radius-4);
-  background: var(--color-accent-10);
+  border-radius: ${uiKitDesignTokens.borderRadius4};
+  background: ${uiKitDesignTokens.colorAccent10};
   box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.25);
-  font-size: var(--font-size-for-navbar-link);
-  line-height: var(--line-height-for-navbar-link);
-  font-weight: var(--font-weight-for-navbar-link);
-  color: var(--color-surface);
+  font-size: ${appKitDesignTokens.fontSizeForNavbarLink};
+  line-height: ${appKitDesignTokens.lineHeightForNavbarLink};
+  font-weight: ${appKitDesignTokens.fontWeightForNavbarLink};
+  color: ${uiKitDesignTokens.colorSurface};
   max-height: ${NAVBAR.itemSize};
   visibility: inherit;
 `;
 
 const TextLink = styled.a`
-  color: var(--color-for-navbar-link);
+  color: ${appKitDesignTokens.colorForNavbarLink};
   text-decoration: none;
   display: flex;
-  padding: var(--spacing-25);
+  padding: ${uiKitDesignTokens.spacing25};
   align-items: center;
   justify-content: center;
 `;
 
 const SupportMenu = styled.div`
-  padding: var(--spacing-10) var(--spacing-30) var(--spacing-20)
-    var(--spacing-30);
-  height: calc(var(--item-size) + var(--spacing-20));
```

Comment:
The differences in the visual regression tests come from this line. 
Before it was 
<img width="258" alt="Screenshot 2024-02-07 at 15 15 02" src="https://github.com/commercetools/merchant-center-application-kit/assets/49066275/206951b7-79c8-499a-9bf5-e51173461cfb">
(this css var doesn't exist)

Now it is
<img width="261" alt="Screenshot 2024-02-07 at 15 13 28" src="https://github.com/commercetools/merchant-center-application-kit/assets/49066275/c970004c-3db7-43d7-ac49-22b15d443b2f">